integrationTests/testProcessorNode.go

```diff
@@ -115,9 +121,28 @@ type TestProcessorNode struct {
 }
 
 // NewTestProcessorNode returns a new TestProcessorNode instance
-func NewTestProcessorNode(maxShards uint32, nodeShardId uint32, txSignPrivKeyShardId uint32, initialNodeAddr string) *TestProcessorNode {
+func NewTestProcessorNode(
+	maxShards uint32,
+	nodeShardId uint32,
+	txSignPrivKeyShardId uint32,
+	initialNodeAddr string,
+	validatorsKeys map[uint32][]*TestKeyPair,
```

Comment:
a new node should have a key pair associated from the validator keys. Currently testPorcessorNode does not have bls keypair. Have to see if it is desirable to have it, as it is only a processor node, without crypto and consensus.

Reply:
used mock here, as not really needed for these tests, in case real signing of blocks is desired, we will have a different setup with bls keys